src/Cameras/babylon.camera.ts

```diff
@@ -207,7 +207,7 @@ module BABYLON {
         /**
          * When set, the camera will render to this render target instead of the default canvas
          */
-        public customDefaultRenderTarget: Nullable<RenderTargetTexture> = null;
+        public renderTarget: Nullable<RenderTargetTexture> = null;

```

Comment:
After giving it some thoughts I think they can be both used actually:
- First case (previous one): Useful if you want multiple views/render per camera
- Second case (this one): used to reroute completely a camera

I guess we will just need to make sure to explain both use cases in the documentation

Reply:
Yea I talked to Seb about this. I think maybe I should call this outputRenderTarget? And ideally customRenderTargets  would be renamed to inputRenderTargets but since I cant do that Ill make sure to document it.